lasagne/layers/conv.py

```diff
@@ -48,7 +49,7 @@ def __init__(self, incoming, num_filters, filter_length, stride=1,
             self.nonlinearity = nonlinearity
 
         self.num_filters = num_filters
-        self.filter_length = filter_length
+        self.filter_size = filter_size
```

Comment:
I think if we're going to accept integers and tuples for the 2D and 3D case, there is no reason to only accept integers for the 1D case. It seems more consistent to use `as_tuple` everywhere imo.

> We could just make it a rule that things that must be tuples are `shape`, and things that can also be scalars are `size`. If we follow that everywhere, this would still count as consistent.

Works for me. But using `shape` everywhere is also fine really. It's not that crucial.